sys-libs/minizip-ng/minizip-ng-3.0.6.ebuild

```diff
@@ -0,0 +1,32 @@
+# Copyright 2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake
+
+DESCRIPTION="Fork of the popular zip manipulation library found in the zlib distribution"
+HOMEPAGE="https://github.com/zlib-ng/minizip-ng"
+SRC_URI="https://github.com/zlib-ng/minizip-ng/archive/refs/tags/3.0.6.tar.gz -> ${P}.tar.gz"
+
+LICENSE="ZLIB"
+SLOT="0"
+KEYWORDS="~amd64"
+
+IUSE="+compat test"
+
+DEPEND="app-arch/bzip2
```

Comment:
I'm surprised it doesn't depend on zlib or zlib-ng. Does it bundle it?

Reply:
I'm not sure whether it is abi compatible or not.
That's why I described loosely, however it is api compatible, so I just updated description in this way.
Anyway this ebuild still is a draft, I just made it quick to be usable by the package I'm targeting.